### ISSUE TITLE
Voting with slider - cosmetic

### ISSUE THREAD

**Issue by 4Ykw** [eSteemApp/esteem#163]
When I tried to vote with 0.01% weight. The message that appears is this one:
![image](https://user-images.githubusercontent.com/6281545/27845608-dc08bdc0-6184-11e7-9174-0e31bb9323de.png)

My suggestion is to also add " or try to increase voting slider percentage" or something like that.

**Comment by feruzm** [comment]
Good idea, error message is coming from blockchain though...but we can add extra explanation if necessary

**Comment by 4Ykw** [comment]
But here we are also translating right? I know it will be a pain to ask for more translation... maybe park it somewhere until there's a good reason to change it.
